students/Steven/lesson09/donor_models.py

```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/env python3
+from operator import itemgetter
+
+class Donor(object):
+    def __init__(self, donor_name, new_donation=None):
+        self.name = donor_name
+        if new_donation is None:
+            self.donation = []
+        else:
+            self.donation = list(new_donation)
+
+    def new_donation(self, new_amount):
+        update_donation = self.donation.append(new_amount)
+        return update_donation
```

Comment:
you don't need to return anything - you have access to the `self.donation` attribute.

besides, `append()` returns `None` so the return here has no purpose